css/components/product_list.css

```diff
@@ -1,116 +1,171 @@
+@import "../util/styled_button.css";
+@import "./modal.css";
+@import "../util/icon_button.css";
+
+/* add product form */
+.btn-container {
+  width: 240px;
+  margin: 10px auto 30px auto;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/**/
+.form-control {
+  margin-bottom: 15px;
+  width: 90%;
+  margin: 0 auto 15px auto;
+  display: flex;
+  justify-content: space-between;
+}
+
 .product-list-container {
-    width: 88%;
-    margin: 0 auto 50px auto;
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
-    gap: 40px 20px;
-  }
-  .product-card-regular {
-    position: relative;
-    justify-self: center;
-    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
-    transition: box-shadow 0.3s;
-    border-radius: 5px;
-    max-width: 260px;
-    height: 300px;
+  width: 88%;
+  margin: 0 auto 50px auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+  gap: 40px 20px;
+}
+.product-card-regular {
+  position: relative;
+  justify-self: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: box-shadow 0.3s;
+  border-radius: 5px;
+  max-width: 260px;
+  height: 300px;
+}
+
+.product-card-regular:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  /*animation: size-up 200ms forwards; -- can't have this one as position fixed for modal doesn't work then*/
```

Comment:
You shouldn't really use `animation` for hover styles. `transition` is your friend.

```css
.product-card-regular { 
  transition: transform 0.3s;
}

.product-card-regular:hover {
  transform: scale(1.1)
}
```